sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -85,14 +85,19 @@ class CheckAnalysis {
 
             cleaned.foreach(checkValidAggregateExpression)
 
+          case _ => // Fallbacks to the following checks
```

Comment:
Because any `Aggregate` operator must match the two case branches above, and will never match those two for unresolved operator and missing input attribute(s).